docs/use cases/README.md

```diff
@@ -1,149 +1,172 @@
+        start
+        :Користувач авторизується
+        в системі;
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
+
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач ознайомлюється 
+        з списком розробників;
+        
+        :Користувач співвідносить
+        завдання та розробників;
+        
+        :Користувач зберігає
+        введену інформацію;
+
+    |Система|
+        :Система записує зміни;
+
+    |Користувач|
+        :Користувач завершує
+        взаємодію;
+        stop;
+
+@enduml
 ```
 
-яка буде відображена наступним чином
+- ID: LEAD_04
 
-<center style="
-    border-radius:4px;
-    border: 1px solid #cfd7e6;
-    box-shadow: 0 1px 3px 0 rgba(89,105,129,.05), 0 1px 1px 0 rgba(0,0,0,.025);
-    padding: 1em;"
->
+```md
 
 @startuml
 
-    right header
-        <font size=24 color=black>Package: <b>UCD_3.0
+    left header
+
+        <font color=000 size=18><b>ID:</b> LEAD_04
+
+        <font color=000 size=16><b>НАЗВА:</b> Перевірити завдання
+        <font color=000 size=16><b>УЧАСНИКИ:</b> Тімлід, Система
+
+        <font color=000 size=16><b>ПЕРЕДУМОВИ:</b> Завдання знаходиться у стані REVIEW
+
+        <font color=000 size=16><b>РЕЗУЛЬТАТ:</b> Перевірене завдання
+
+        <font color=000 size=16><b>ВИКЛЮЧНІ СИТУАЦІЇ:</b> Відсутні
+
+        <font color=000 size=16><b>ОСНОВНИЙ СЦЕНАРІЙ:</b>
+
     end header
 
-    title
-        <font size=18 color=black>UC_8. Редагувати конфігурацію порталу
-        <font size=16 color=black>Діаграма прецедентів
-    end title
-
-
-    actor "Користувач" as User #eeeeaa
-    
-    package UCD_1{
-        usecase "<b>UC_1</b>\nПереглянути список \nзвітів" as UC_1 #aaeeaa
-    }
-    
-    usecase "<b>UC_1.1</b>\nЗастосувати фільтр" as UC_1.1
-    usecase "<b>UC_1.2</b>\nПереглянути метадані \nзвіту" as UC_1.2  
-    usecase "<b>UC_1.2.1</b>\nДати оцінку звіту" as UC_1.2.1  
-    usecase "<b>UC_1.2.2</b>\nПереглянути інформацію \nпро авторів звіту" as UC_1.2.2
-    
-    package UCD_1 {
-        usecase "<b>UC_4</b>\nВикликати звіт" as UC_4 #aaeeaa
-    }
-    
-    usecase "<b>UC_1.1.1</b>\n Використати \nпошукові теги" as UC_1.1.1  
-    usecase "<b>UC_1.1.2</b>\n Використати \nрядок пошуку" as UC_1.1.2
-    usecase "<b>UC_1.1.3</b>\n Використати \nавторів" as UC_1.1.3  
-    
-    
-    
-    User -> UC_1
-    UC_1.1 .u.> UC_1 :extends
-    UC_1.2 .u.> UC_1 :extends
-    UC_4 .d.> UC_1.2 :extends
-    UC_1.2 .> UC_1.2 :extends
-    UC_1.2.1 .u.> UC_1.2 :extends
-    UC_1.2.2 .u.> UC_1.2 :extends
-    UC_1 ..> UC_1.2.2 :extends
-    
-    
-    UC_1.1.1 -u-|> UC_1.1
-    UC_1.1.2 -u-|> UC_1.1
-    UC_1.1.3 -u-|> UC_1.1
-    
-    right footer
-        Аналітичний портал. Модель прецедентів.
-        НТУУ КПІ ім.І.Сікорського
-        Киів-2020
-    end footer
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
 
-@enduml
+        :Користувач відкриває вікно
+        з усіма завданнями;
```

Comment:
Would be better kinda:

1. User clicks the button "Tasks"
2. The system is sending task list to user

...